forte/data/data_pack.py

```diff
@@ -824,7 +823,7 @@ def get_data(
             A data generator, which generates one piece of data (a dict
             containing the required entries, fields, and context).
         """
-        context_type_: Type[Annotation]
+        context_type_: Union[Type[Annotation], Type[AudioAnnotation]]
```

Comment:
I realize we haven't reflected the changes in the docstring. We need to add/edit the examples, and make the docstring accurate

Reply:
Consider adding example and usage of this to here: https://github.com/asyml/forte/blob/master/docs/data_pack.md